src/discord.ts

```diff
@@ -128,10 +128,11 @@ export async function buildClient(shard: number, shardCount: number): Promise<Cl
       const response = typeof command.response === 'function' ? await command.response(msg, guildConfig) : command.response;
 
       if (typeof response === 'string') {
-        msg.channel.send(response);
+        // We might want preventExceed here instead
+        msg.channel.send(response, {split: true});
```

Comment:
Depending on what we prefer we may want to either split the message, or simply cut it off.
1. wall of text (over 2000 chars)
2. lost info